python/aibrix_kvcache/pyproject.toml

```diff
@@ -42,8 +42,13 @@ tqdm = ">=4.67.1,<5.0.0"
 cityhash = "^0.4.8"
 redis = "^6.0.0"
 fakeredis = "^2.28.1"
-infinistore = "^0.2.35"
+infinistore = { version = "^0.2.35", optional = true }
```

Comment:
We should ask them to release a stable version with `--hint-gid-index` so we can bump the version here.